docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -52,7 +52,46 @@ and :ref:`our facility choice module documentation <2024_vivarium_mncnh_portfoli
 2.0 Module Input and Output Data
 ++++++++++++++++++++++++++++++++
 
-2.1 Module Description 
+2.1 Module Inputs
+-----------------
+
+.. list-table:: Module required inputs
+  :header-rows: 1
+
+  * - Input
+    - Source 
+    - Definition
+    - Application
+    - Note
+  * - Pregnancy term duration 
+    - :ref:`Pregnancy I <2024_vivarium_mncnh_portfolio_pregnancy_module>`
+    -
+    - Partial term pregnancies by default should be assigned the probability value of ANCfirst.
```

Comment:
This sounds confusing to me -- we're not really "assigning a probability value" to the pregnancies or the simulants.

Maybe it would be better to say something like, "Determines which set of probabilities to use for the ANC exposure categories. Full term pregnancies can be assigned any of the values A, B, C, or D, whereas we assume partial term pregnancies can only attend ANC in the first trimester, so only categories B and D have nonzero probability. See exposure probability tables below."

But maybe it doesn't need to be so wordy, or maybe some of this can go in the Note column instead of the Application column.